src/main/java/org/variantsync/diffdetective/diff/result/CommitDiffResult.java

```diff
@@ -19,9 +20,10 @@ public record CommitDiffResult(Optional<CommitDiff> diff, List<DiffError> errors
      * @return A failure result that was caused by the given error.
      */
     public static CommitDiffResult Failure(DiffError error, String message) {
+        Logger.debug("[CommitDiffResult::Failure] {}", message);
```

Comment:
The `"[CommitDiffResult::Failure]"` should be inserted automatically by Log4J so it is redundant here, isn't it?

Reply:
Good catch. Yes you're right. I removed this prefix now.